client/my-sites/upgrades/checkout-thank-you/plans-page-your-plan.js

```diff
@@ -0,0 +1,96 @@
+/**
```

Comment:
@mtias, I was thinking about that.
But what about CSS ?
I dont want to duplicate all of these classes, because I later want to improve this page and I kinda want to avoid huge refactors of yet another thing.

So, If I move it to `my-sites/plan` or something similar, will you let me keep current CSS ?
If not, Im not moving it anywhere :)